openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -389,7 +389,31 @@
         }
 
         .copy, .icon {
-          color: $white-t;
+          color: $white;
+        }
+      }
+
+      button {
+        opacity: 1;
+      }
+    }
+
+    // STATE: in-progress
+    &.is--in-progress {
+
+      .step__counter:before {
+        color: $uxpl-primary-base;
```

Comment:
I will switch to using $action-primary-color, which I think conveys the intent here.